internal/command/postgres/import.go

```diff
@@ -95,6 +104,17 @@ func runImport(ctx context.Context) error {
 		return fmt.Errorf("The target app must be a Postgres app")
 	}
 
+	machines, err := flapsClient.ListActive(ctx)
+	if err != nil {
+		return fmt.Errorf("could not retrieve machines: %w", err)
+	}
+
+	if len(machines) == 0 {
+		return fmt.Errorf("no machines are available on this app %s", appName)
+	}
+	// It doesn't matter which machine we choose
+	machineID := machines[0].ID
```

Comment:
Just a thought: I think it's possible to pick the primary with [`machinesNodeRoles`](https://github.com/superfly/flyctl/blob/0bb265d6a68309fe8e65624a4e6d5f75f44ebd1d/internal/command/postgres/postgres.go#L121) or something similar.

Reply:
oh shoot i didn't know about that! neat!